budgetApp1/budgetApp1/Assets.xcassets/profileIcon.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "IMG_8955 2.jpg",
```

Comment:
Please change the actual asset name to be more specific

Reply:
I removed it because I don't use it